woocommerce/woocommerce-gutenberg-products-block
# Add collection filters block

## What

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/10711

This PR adds a new block called `Collection Filters` block. It will serve as a wrapper container for all active filters we have. It is also a dependent of the `Product Collection` block.

At this point, the block is just a simple block act as the wrapper for other blocks, we want to merge this PR to trunk to make develop other inner block easier. This PR won't close its connected issue as we haven't fully implemented the block yet.

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. On a post/page add the `Product Collection` block.
2. Using the block inserter within the `Product Collection` block, try to add the `Collection Filters` block. Note that it will not work outside of the `Product Collection` Block.
3. You should now be able to select the filter you want to use.
4. Randomly choose one and ensure it indeed adds the filter to the page and save.
5. Go to the frontend and ensure you see the filter you've added within the `Product Collection` block.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
